src/api/services/login-service.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Created by oswaldogallango on 7/11/16.
+ */
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs/Rx';
+import {ApiRoot} from '../persistence/ApiRoot';
+import {Http, RequestMethod} from '@angular/http';
+import {CoreWebService} from '../services/core-web-service';
+
+/**
+ * This Service get the server configuration to display in the login component
+ * and execute the login and forgot password routines
+ */
+@Injectable()
+export class LoginService extends CoreWebService {
+
+    private serverInfo: Array<any>;
+    private userAuthURL: string;
+    private serverInfoURL: string;
+    private recoverPasswordURL: string;
+    private logoutURL: string;
+
+    private lang: string = 'en';
+    private country: string = 'US';
+
+    constructor(_apiRoot: ApiRoot, _http: Http) {
+        super(_apiRoot, _http);
+
+        this.userAuthURL = `${_apiRoot.baseUrl}api/v1/authentication`;
```

Comment:
This is not a good idea because every URL could be using a different version of the rest apis.